packages/reports/addon/components/filter-values/multi-value-input.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   <FilterValues::MultiValueInput
+ *     @filter={{this.filter}}
+ *     @onUpdateFilter={{this.update}}
+ *   />
+ */
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import FilterFragment from 'navi-core/models/bard-request-v2/fragments/filter';
+import { tracked } from '@glimmer/tracking';
+
+interface MultiValueInputArgs {
+  filter: FilterFragment;
+  onUpdateFilter(changeSet: Partial<FilterFragment>): void;
+}
```

Comment:
Can we use ` args-interface.ts` instead?

Reply:
Can you explain this further?

Reply:
```ts
import Args from './args-interface';
```

Reply:
and then would I do interface MultiValueInputArgs extends Args?

Reply:
Similar to what was done here: https://github.com/yahoo/navi/pull/1057/files#diff-87c52ce7e472e1f711c579c2a929aebfR8